<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QJlogic</class>
 <widget class="QMainWindow" name="QJlogic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>闭塞分区确认无车占用</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>520</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>闭塞分区确认无车占用</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>无车验证</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>无车执行</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>车站</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>140</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>闭塞分区</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>361</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 250, 250);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>命令反馈结果</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>区间逻辑状态总解锁</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>区间方向口</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>车站</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>140</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>总解锁执行</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>总解锁验证</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>命令反馈结果</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_9">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>361</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 250, 250);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>区间逻辑检查开启</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开启执行</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开启验证</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>区间方向口</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>车站</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_7">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>140</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>命令反馈结果</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_10">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>361</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 250, 250);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>区间逻辑检查关闭</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭执行</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭验证</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>区间方向口</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>车站</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_19">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>140</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_20">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>命令反馈结果</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_11">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>361</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 250, 250);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
